benchmark/autoencoder/aa.py

```diff
@@ -13,7 +13,7 @@
 
 from theano.tensor import blas, opt
 
-# numpy: aa_numpy.py
```

Comment:
Please do not blindly auto-replace "numpy" by "np". Here, we are talking about a numpy implementation and a filename, not about the module name.
You could replace "numpy:" in that comment by "NumPy" if you want, but "np" is not clear enough.
Moreover, the name of the file is still "aa_numpy.py", not "aa_np.py".